terraform/fargate-examples/sqs-dynamic-target-tracking/versions.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_version = ">= 1.1"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.43"
```

Comment:
Can we change this to version = ">= 5.0" ?

Reply:
done